pkgs/by-name/le/leaf-arpa2/package.nix

```diff
@@ -0,0 +1,50 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitLab,
+  cmake,
+  arpa2cm,
+  quickder,
+  quickmem,
+  quick-sasl,
+  lillydap,
+  nix-update-script,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  # leaf is already used
+  pname = "leaf-arpa2";
```

Comment:
Then how about we use the `arpa2-*` namespace for all `arpa2` derivation? So this would become `arpa2-leaf`.

Reply:
I considered that but if you are looking for leaf I think one would be more likely to find `leaf-arpa2` rather than `arpa2-leaf` in search.nixos.org or similar? I'm not sure. Let me know what you think. I am good with either.

Reply:
IMO, there is a limited number of derivations named `leaf` something, so this would show up in either case, but looking at this from another angle, if we adopt the namespace then searching for `arpa2` will give you all derivations related to this project.

Furthermore, this could also allow us to filter for related derivations more easily by just checking if their `pname` prefix matches the `arpa2` namespace.

Reply:
Makes sense. Will fix. Thanks!